### ISSUE TITLE
idm was registered with a fake serial number message!

### ISSUE THREAD

**Issue by ahmetcanisik** [lstprjct/IDM-Activation-Script#32]
Hello, I am getting the error that idm is registered with a fake serial number. It worked fine until now but now I'm getting this error.

**Comment by Hsn5** [comment]
use freeze trial instead

**Comment by ahmetcanisik** [comment]
Yes, I have tested that the freeze trial option always works, but I am wondering why the recording option does not work and I wonder if this will be fixed?
